src/transformers/modeling_reformer.py

```diff
@@ -514,6 +514,15 @@ def _hash_vectors(self, vectors, num_hashes):
 
                 cur_product = cur_product * bucket_factor
 
+        if attention_mask is not None:
```

Comment:
If an `attention_mask` is given an extra bucket is added and buckets assignments for positions that are padded are given this new bucket. This code is new and thus changes the integration tests.